lib/daru/dataframe.rb

```diff
@@ -925,8 +926,9 @@ def all? axis=:vector, &block
     #
     # @param [Fixnum] quantity (10) The number of elements to display from the top.
     def head quantity=10
-      return Daru::DataFrame.new [self.row[0].to_a].transpose,
-        index: [@index.at(0)], order: @vectors if quantity == 1
+      return Daru::DataFrame.rows [row[0]],
+        index: [@index.at(0)],
+        order: @vectors if quantity == 1
       self[0..(quantity-1), :row]
```

Comment:
Yeah it looks better. BTW I wonder why don't Ruby give up `end` and follow indentation like Python to differentiate if statement lies under `if/else` or not.